metaserver/src/test/scala/tests/search/SymbolIndexTest.scala

```diff
@@ -51,28 +52,70 @@ object SymbolIndexTest extends MegaSuite {
         line: Int,
         column: Int,
         expected: String
-    ): Unit = {
-      val term = indexer.findSymbol(path, line, column)
+    ): Option[Symbol] = {
+      val symbol = indexer.findSymbol(path, line, column)
+      Predef.assert(
+        symbol.isDefined,
+        s"Symbol not found at $path:$line:$column."
+      )
+      assertNoDiff(symbol.get.syntax, expected)
       Predef.assert(
-        term.isDefined,
-        s"Symbol not found at $path:$line:$column. Did you run scalametaEnableCompletions from sbt?"
+        indexer.symbolIndexer.get(symbol.get).isDefined,
+        s"Symbol ${symbol} is not found in the index. Did you run scalametaEnableCompletions from sbt?"
       )
-      assertNoDiff(term.get.symbol, expected)
+      symbol
+    }
+
+    def assertSymbolDefinition(
+        expected: String
+    )(
+        symbol: Symbol
+    ): Unit = {
+      val data = indexer.definitionData(symbol)
       Predef.assert(
-        term.get.definition.isDefined,
-        s"Definition not found for term $term"
+        data.isDefined,
+        s"Definition not found for term ${symbol}"
       )
+      assertNoDiff(data.get.symbol, expected)
     }
 
-    "fallback" - {
+    "find symbol and its definition" - {
```

Comment:
I keep the names here short so I can `testOnly tests.search.SymbolIndexTest.fallback`, we can add a comment if think it's unclear

Reply:
Fixed in 3548ab933e71621b13af599261d5da2f16e65567